client/cypress.config.ts

```diff
@@ -2,9 +2,9 @@ import { defineConfig } from 'cypress';
 
 export default defineConfig({
   e2e: {
-    setupNodeEvents(on, config) {
+    setupNodeEvents: (on, config) => {
```

Comment:
The configuration file is automatically generated, but our linter doesn't like the formatting it uses. I updated the syntax for the method here just for the linter to calm down.